content/reference/config.md

```diff
@@ -546,6 +548,68 @@ dbs:
           - age1xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 ```
 
+## Cost Considerations
+
+Understanding the cost implications of Litestream configuration is important for
+managing cloud storage expenses. The primary cost drivers vary by configuration:
+
+### Sync Interval Costs
+
+The `sync-interval` setting primarily affects **PUT request costs**, not storage costs.
+Litestream only uploads frames when the database changes, but frequent intervals
+mean more frequent requests when changes occur.
+
+**PUT Request Cost Examples (AWS S3 pricing: $0.000005 per request):**
+
+- `sync-interval: 1s` with constant writes: ~2,592,000 requests/month = **$12.96/month**
+- `sync-interval: 10s` with constant writes: ~259,200 requests/month = **$1.30/month**  
+- `sync-interval: 1m` with constant writes: ~43,200 requests/month = **$0.22/month**
+
+_Note: Actual costs depend on your database write patterns. Litestream batches
+writes by time interval, so costs scale with write frequency._
+
+### Storage vs Request Costs
+
+**Request Costs (Primary Driver):**
+
+- PUT operations for uploading frames and snapshots
+- GET operations for restores and validation
+- LIST operations for cleanup and maintenance
+
+**Storage Costs (Usually Minimal):**
+
+- LTX frame files (SQLite page updates)
```

Comment:
You can just call these "LTX files". The frames are each page and its metadata within there.